### ISSUE TITLE
Test on GPU in CI

### ISSUE THREAD

**Issue by uellue** [LiberTEM/sparseconverter#47]
Right now we only cover about half the code in CI since GPU support is missing.

Discussion with @sk1p: Possibly a target for running at JSC? Also allows running the benchmarks

**Comment by sk1p** [comment]
> Discussion with [@sk1p](https://github.com/sk1p): Possibly a target for running at JSC? Also allows running the benchmarks

Good idea! We should reserve a day or so for setting up GPU CI for the various relevant packages (LiberTEM{,holo,blobfinder,live} and sparseconverter?), which is then also a good starting point for continuous benchmarking.
